xmind/tests/base.py

```diff
@@ -50,5 +53,24 @@ def side_effect_function(*a, **kw):
         _mock.side_effect = _side_effect
 
         self._patchers.append(property_name)
-        self.LOGGER.debug("Property '%s' for function '%s' has been set", property_name, function_name)
-        return _mock
\ No newline at end of file
+        self.LOGGER.debug(
+            "Property '%s' for function '%s' has been set", property_name, function_name)
+        return _mock
+
+    def _test_method_by_excessive_parameters(self, pair, _element):
```

Comment:
@vlad-shulika , @mykola-pysarev 
**Please check it carefully**
I introduced this method in base class just to be able to do excessive parameters check.
Please check test_excessive_parameters test in test_element or test_document classes to get understanding how it used there